fuel/app/themes/default/partials/catalog/media.php

```diff
@@ -62,6 +61,12 @@
 				</span>
 				<span class='file-date'>
 					{{file.created}}
+					<br>
+					<button ng-class="file.is_deleted == 0 ? 'delete-btn green' : 'delete-btn orange'"
+						ng-click='deleteAsset(file)'>
+						<span ng-if='file.is_deleted == 0'>DELETE</span>
+						<span ng-if='file.is_deleted == 1'>DELETED</span>
```

Comment:
Given my comments in the review generally, we should probably change the word `DELETED` to be something more directly tied to the action the button will perform in this state - `Restore` or `Undelete` seems more appropriate.